ethsigner/signer/multiplatform/src/test/java/tech/pegasys/ethsigner/signer/multiplatform/SigningMetadataTomlConfigLoaderTest.java

```diff
@@ -187,11 +200,38 @@ void loadAvailableConfigsReturnsAllValidMetadataFilesInDirectory() {
 
     // duplicate files are loaded at this stage since addresses aren't checked until signers are
     // created
-    final Collection<FileBasedSigningMetadataFile> metadataFiles =
+    final Collection<SigningMetadataFile> metadataFiles =
         loader.loadAvailableSigningMetadataTomlConfigs();
 
     assertThat(metadataFiles).hasSize(5);
     assertThat(metadataFiles)
         .containsOnly(metadataFile1, metadataFile2, metadataFile3, metadataFile4, metadataFile5);
   }
+
+  @Test
+  void azureConfigIsLoadedIfAzureMetaDataFileInDirectory() {
+    final String metaDataFilename = "azureconfig.toml";
+    final Path newMetadataFile = configsDirectory.resolve(metaDataFilename);
```

Comment:
nit: elsewhere we have used metadata not metaData

Reply:
deal - gone with metadata across the board.